### PR TITLE
Update rmdir usage

### PR DESCRIPTION
**Pull request by frikiluser**
Please consider merging this usage/man change to document recent **breaking changes** on rmdir tool.

Related to this bug: https://debbugs.gnu.org/cgi/bugreport.cgi?bug=39364
And this commit: https://github.com/coreutils/coreutils/commit/1f443fe57291918ba245b2ffb7cdc07e64630af4

Alternatively revert the linked commit.
Regards,

### CODE REVIEW

**Comment by frikiluser** [comment]
Current version:
```
% rmdir /
rmdir: failed to remove '/': Device or resource busy
% echo $?
1
% rmdir / --ignore-fail-on-non-empty
% echo $?
0
```

Older versions (tested on 8.30):
```
% rmdir / --ignore-fail-on-non-empty
rmdir: failed to remove '/'
% echo $?
1
```

**Comment by caelwithcats** [comment]
@frikiluser I wouldn't want to delete my root filesystem

**Comment by masterneko** [comment]
```
rmdir /  --ignore-fail-on-non-empty
```
**DESTRUCTION!**

**Comment by frikiluser** [comment]
> ```
> rmdir /  --ignore-fail-on-non-empty
> ```
> 
> **DESTRUCTION!**

Not really!
As it can't be removed for multiple reasons.

One reason already mentioned: ``` rmdir: failed to remove '/': Device or resource busy ```

More reasons:
**Coreutils 8.30**
```
% sudo rm -rf /tmp/rmdir ; sudo mkdir /tmp/rmdir/a/b -pv ; echo $UID ; rmdir --ignore-fail-on-non-empty /tmp/rmdir/a ; echo $?
mkdir: created directory '/tmp/rmdir'
mkdir: created directory '/tmp/rmdir/a'
mkdir: created directory '/tmp/rmdir/a/b'
1000
rmdir: failed to remove '/tmp/rmdir/a'
1
```

**Coreutils 8.32**
```
sudo rm -rf /tmp/rmdir ; sudo mkdir /tmp/rmdir/a/b -pv ; echo $UID ; rmdir --ignore-fail-on-non-empty /tmp/rmdir/a ; echo $?
mkdir: created directory '/tmp/rmdir'
mkdir: created directory '/tmp/rmdir/a'
mkdir: created directory '/tmp/rmdir/a/b'
1000
0
```

I don't have the enough history knowledge to request a revert. My proposal here is to update the command description according to new behavior to avoid confusions. In my understanding the word **solely** should be removed.

Thanks for your attention.

**Comment by pixelb** [comment]
The history of this option is confusing.

It originally behaved as you desired, but then was adjusted to support rmdir --parents, to essentially remove as much of a hierarchy as possible, without erroring as we hit more protected non empty parent dirs.  That suggestion was incorrectly referenced in the commit, but came from
https://lists.gnu.org/archive/html/bug-coreutils/2008-01/msg00283.html

So in summary your doc change is consistent with the current behavior, though we also need to adjust the texinfo.

I'll fix this up shortly. Thanks

**Comment by pixelb** [comment]
Addressed in https://github.com/coreutils/coreutils/commit/dd614d6011f64e06852adc532e7357de59f4bac3
